services/sso/src/controllers/auth/loginController.ts

```diff
@@ -10,7 +10,7 @@ export const loginController: Middleware = async (ctx: Context) => {
   const { dni, password }: UserLogin = ctx.request.body
   const dniToUpperCase = dni.toUpperCase()
   const userResult = await client.query(
-    'SELECT id, password, status FROM "user" WHERE dni = $1 AND deleted_at IS NULL',
+    'SELECT id, password, status, FROM "user" WHERE dni = $1 AND deleted_at IS NULL',
```

Comment:
This comma is what causes all the tests with 'login' to be failing, remove it:
```suggestion
    'SELECT id, password, status FROM "user" WHERE dni = $1 AND deleted_at IS NULL',
```